### PR TITLE
Ag 10044 design refactor

### PR DESCRIPTION
**Pull request by markdurrant**
Massive design refactor.

### CODE REVIEW

File: packages/ag-charts-website/src/components/release-notes/Collapsible.module.scss [hunk 1]
```diff
@@ -38,33 +31,32 @@
...
+    border-bottom-left-radius: var(--radius-sm);
     border-right: $border;
     border-bottom: $border;
     border-left: $border;
     overflow: hidden;
 
     &:not(.notExpandable) > div {
         overflow: hidden;
-        //padding-bottom: $size-7;
+        //padding-bottom: $spacing-size-16;
```
Comment (taktran) [inline packages/ag-charts-website/src/components/release-notes/Collapsible.module.scss]: Maybe can delete this?

File: packages/ag-charts-website/src/components/icon/Icon.module.scss
Comment (kylerphillips) [inline packages/ag-charts-website/src/components/icon/Icon.module.scss:1]: I'd probably expect something like `color-fg-primary` here

File: packages/ag-charts-website/src/design-system/_layout.scss [hunk 2]
```diff
@@ -4,3 +4,100 @@
...
+.layout-grid {
+    display: flex;
+    flex-wrap: wrap;
+    gap: var(--layout-gap);
+    max-width: min(calc(100% - var(--layout-gap) * 2), var(--layout-max-width));
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.column-1-4 {
```
Comment (taktran) [inline packages/ag-charts-website/src/design-system/_layout.scss:17]: Where do we use this? Wouldn't we be able to do this with flexbox or grid?

File: packages/ag-charts-website/src/components/search/gridSearch.module.scss
Comment (kylerphillips) [inline packages/ag-charts-website/src/components/search/gridSearch.module.scss:1]: Any reason we didn't change `8px` to` $spacing-size-2`?

Comment (markdurrant) [reply packages/ag-charts-website/src/components/search/gridSearch.module.scss:1]: No idea. 

I'll update that now .

**Review by taktran — APPROVED** [review]
Epic work 👏🏼  My fav part was the removal of `!important`s 😂 

Just some minor comment

File: packages/ag-charts-website/src/features/pipeline/pipelineChangelog.module.scss [hunk 3]
```diff
@@ -60,10 +60,10 @@
 }
 
 .fontClass {
-    font-size: 15px !important;
+    font-size: 15px;
```
Comment (kylerphillips) [inline packages/ag-charts-website/src/features/pipeline/pipelineChangelog.module.scss:63]: Should we bump this up to `text-base`?

**Review by kylerphillips — APPROVED** [review]
Left a couple of tiny comments throughout, overall LGTM 🚀

**Pull request merged**